test/api/v2/room_query_test.exs

```diff
@@ -0,0 +1,184 @@
+defmodule RoomQueryTest do
+  @moduledoc """
+  Test absinthe queries on rooms
+  """
```

Comment:
TODO: Create full queries for graphql types. We can use https://github.com/devonestes/assertions/blob/master/lib/assertions/absinthe.ex